File: parent/pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillGoodsService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.seckill.SeckillGoods;
import entity.PageResult;

public interface SeckillGoodsService {
    void add(SeckillGoods seckillGoods);

    PageResult search(Integer page, Integer rows, SeckillGoods seckillGoods);

    void updateStatus(Long[] ids, String status);
}

File: parent/pinyougou-web-shop/src/main/webapp/js/service/seckillGoodsService.js
app.service('seckillGoodsService',function($http){
    //增加
    this.add=function(entity){
        return  $http.post('../seckillGoods/add.do',entity );
    }

    //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../seckillGoods/search.do?page='+page+'&rows='+rows,searchEntity);
    }

});